.eslintrc.cjs

```diff
@@ -18,6 +19,7 @@ module.exports = {
 		},
 		{
 			extends: [
+				"plugin:jsdoc/recommended",
```

Comment:
Adding emphasis:

> chore: Use eslint-plugin-jsdoc's **recommended-typescript-error** ruleset

Close but not quite the right ruleset 😄. Search for `typescript` in https://github.com/gajus/eslint-plugin-jsdoc#configuration.